soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -187,6 +187,7 @@ static bool SetPosHandler(std::shared_ptr<Ship::Console> Console, const std::vec
 }
 
 static bool ResetHandler(std::shared_ptr<Ship::Console> Console, std::vector<std::string> args) {
+    SaveManager::Instance->ThreadPoolWait();
```

Comment:
If `ThreadPoolWait()` works in the `OnExitGame` hook, then this can also be removed thanks to the execute hook below.

Reply:
You are correct on that one.